addons/account/data/mail_template_data.xml

```diff
@@ -45,6 +45,10 @@
                 Please use the following communication for your payment: <span style="font-weight:bold;" t-out="object.payment_reference or ''">INV/2021/05/0005</span>.
             </t>
         </t>
+        <t t-if="hasattr(object, 'timesheet_count') and object.timesheet_count">
+            <br /><br />
+            PS: you can review your timesheets <a t-att-href="'my/timesheets?search_in=invoice&amp;search=%s' % object.name">from the portal.</a>
+        </t>
```

Comment:
Hello @odoo/rd-accounting 👋🏼 

FYI, we added that in the email template used when the user clicks on `Send & Print` button in invoice form view. 🙂

Reply:
Why is this not done in `sale_timesheet`, where the field `timesheet_count` is added instead of this dirty `hasattr`?

Reply:
> Why is this not done in `sale_timesheet`, where the field `timesheet_count` is added instead of this dirty `hasattr`?

We cannot extend the email template.
cc @xavierbol

Reply:
Also, how does this work if the invoice has nothing related to timesheets?
As soon as `sale_timesheet` is installed, we see this additional info?

Reply:
> We cannot extend the email template.

I understand it might not be the most elegant, but it is possible to do in a post init hook for instance, or with a `<function/>` tag in xml.

But it might not be better than this indeed.

Reply:
With your suggestion, I guess we will also have to do `uninstall_hook`. 😅 😬